lib/app-tray.js

```diff
@@ -45,10 +42,20 @@ exports.init = function (gme) {
     type: 'separator'
   }));
   trayMenu.append(new MenuItem({
+    label: 'Show / Hide Google Music Electron',
+    click: gme.onTrayClick
+  }));
```

Comment:
Ah, k. I chose to keep "Exit" isolated to avoid accidental misclicks. For me, hide/show is a common action and I don't want my finger to slip and accidentally exit.